jenkins/FullBuild

```diff
@@ -46,9 +46,10 @@ pipeline {
     stage('Artifact') {
       steps {
         sh 'rm -rf artifacts || true'
-        sh 'mkdir -p artifacts'
+        sh 'mkdir -p artifacts/pkg_dbg'
         sh 'mv src/tmp/release/* artifacts/'
-        }
+        sh 'cp src/tmp/pkgdir/*dbg* artifacts/pkg_dbg/'
```

Comment:
I think it would be useful to download the complete pkg directory as well in case someone wants to just test the new package instead of trying out the complete build ?

Reply:
Oh I like that idea.

Reply:
OK, let's go for it.

Reply:
done